### PR TITLE
Exclude include

### PR DESCRIPTION
**Pull request by Enriquesuarezgtte**

### CODE REVIEW

File: src/test/LoginTest.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	
+	@Test
+	public void webLoginCarLoan() {
+		System.out.println("weblogin");
+	}
+	
+	@Test
+	public void mobileLoginCarLoan(){
+		System.out.println("mobileLogin");
+
```
Comment (yesid-lopez) [inline src/test/LoginTest.java]: delete whitespace

File: src/test/LoginTest.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	}
+	
+	@Test
+	public void loginAPICarLoan() {
+		// Rest API Automation
```
Comment (yesid-lopez) [inline src/test/LoginTest.java]: delete comment

File: src/test/LoginTest.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+		System.out.println("APIlogin");
+
```
Comment (yesid-lopez) [inline src/test/LoginTest.java]: delete whitespaces

File: testng.xml [hunk 4]
```diff
@@ -1,9 +1,37 @@
...
+  	</methods>
+  	</class>
+  	</classes>
+  </test>
+      <test name = "Basic 2 package Loan" >
+  	<packages>
+  		<package name="test"></package>
+  	</packages>
+  </test>
+</suite> <!-- Suite -->
```
Comment (yesid-lopez) [inline testng.xml]: is necessary the Suite comment?

Comment (Enriquesuarezgtte) [reply testng.xml]: No, it is not necessary

File: testng.xml [hunk 5]
```diff
@@ -1,9 +1,37 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
-<suite name="Loan Department">
-  <test thread-count="5" name="Personal Loan0">
-    <classes>
-      <class name="test.Basics"/>
-    </classes>
+<suite name="Suite">
+
```
Comment (yesid-lopez) [inline testng.xml]: delete whitespace

File: testng.xml [hunk 6]
```diff
@@ -1,9 +1,37 @@
...
+  <test name = "Basic Loan" >
+  	<classes>
+  	<class name="test.Basics">
+  	<methods>
+  		<include name="Demo2"></include>
+  	</methods>
+  	</class>
+  	</classes>
+  </test>
+    <test name = "Basic 2 Loan" >
```
Comment (yesid-lopez) [inline testng.xml]: indent code for this file

Comment (Enriquesuarezgtte) [reply testng.xml]: done

File: testng.xml (hunk 5 above)
Comment (Enriquesuarezgtte) [reply testng.xml]: done. Deleted more whitespaces in other files

**Review by yesid-lopez — APPROVED** [review]

**Pull request merged**